Telerik.JustMock/Expectations/NonPublicExpectation.cs

```diff
@@ -29,211 +29,9 @@ JustMock Lite
 
 namespace Telerik.JustMock.Expectations
 {
-	internal sealed class NonPublicExpectation : INonPublicExpectation
+	internal sealed class NonPublicExpectation : NonPublicExpectationBase, INonPublicExpectation
```

Comment:
I add this comment to clearly mark inheritance from NonPublicExpecatationBase which I find to be deviating from "IS A" OOP design principle.

Reply:
@lyubomir-rusev, fixed with [595ded1](https://github.com/telerik/JustMockLite/pull/42/commits/595ded10a032d8e33576643110280741ea96a82d)